prow/Makefile

```diff
@@ -30,6 +30,9 @@ update-config: get-cluster-credentials
 deploy: get-cluster-credentials
 	kubectl apply -f ./cluster/
 
+deploy-velodrome: get-cluster-credentials
```

Comment:
Make this autodeploy in a follow up?

Reply:
Will add it to [job](https://github.com/istio/test-infra/pull/2226/files#diff-fbde41226949bfb348b691c6d52c372cR25) as soon as #2226 merges (to avoid conflict).